themes/_core/js/checkout-address.js

```diff
@@ -32,8 +36,68 @@ export default function () {
     prestashop.emit('editAddress');
   });
 
-  $('#delivery-addresses, #invoice-addresses input[type=radio]').on('click', function () {
+  $('#delivery-addresses input[type=radio], #invoice-addresses input[type=radio]').on('click', function () {
     $('.address-item').removeClass('selected');
     $('.address-item:has(input[type=radio]:checked)').addClass('selected');
+
+    switchConfirmAddresssesButtonState();
```

Comment:
Before, you were always setting the state to enabled. Now it toggles it alternatively on and off. Are you sure this is what you intended?